tests/aftu/test_compare_graphs.py

```diff
@@ -26,42 +30,53 @@
 # NOTE: we need to set VLLM_ENABLE_V1_MULTIPROCESSING=0 otherwise this
 # mock will not propagate to the child process of the model runner
 def mock_get_mask_dtype(mocker: MockerFixture):
-
-    mocker.patch.object(SpyreCausalLM,
-                        "get_mask_dtype",
-                        return_value=torch.float32)
+    mocker.patch.object(SpyreCausalLM, "get_mask_dtype", return_value=torch.float32)
 
 
 @pytest.mark.spyre
 @pytest.mark.cb
-def test_compare_graphs_cb(model: ModelInfo, max_num_seqs: int,
-                           max_model_len: int, monkeypatch: pytest.MonkeyPatch,
-                           mocker: MockerFixture):
+def test_compare_graphs_cb(
+    model: ModelInfo, max_num_seqs: int, max_model_len: int, monkeypatch: pytest.MonkeyPatch, mocker: MockerFixture
+):
     """Test that the spyre worker correctly outputs
     continuous batches of requests by comparing to HF"""
 
     # AFTU
     script_dir = get_aftu_script_dir()
     if script_dir is None:
-        pytest.skip("aiu-fms-testing-utils is required "
-                    "and is not installed to run this test")
+        pytest.skip("aiu-fms-testing-utils is required and is not installed to run this test")
 
     model_path = get_model_path(model)
 
-    attn_type = 'paged_fp8' if model.is_quantized \
-        else 'paged'
+    attn_type = "paged_fp8" if model.is_quantized else "paged"
 
     if model.is_quantized:
         mock_get_mask_dtype(mocker)
 
     inference_py_args = [
-        sys.executable, "scripts/inference.py", "--architecture",
-        "hf_pretrained", "--model_path", model_path, "--tokenizer", model_path,
-        "--unfuse_weights", "--device_type", "aiu", "--compile",
-        "--cast_bf16_to_fp16", "--compile_dynamic", "--min_pad_length", "64",
-        "--max_new_tokens", "5", "--batch_size",
-        str(max_num_seqs), "--compile_dynamic_sendnn", "--attention_type",
-        attn_type
+        sys.executable,
+        "scripts/inference.py",
+        "--architecture",
+        "hf_pretrained",
+        "--model_path",
+        model_path,
+        "--tokenizer",
+        model_path,
+        "--unfuse_weights",
+        "--device_type",
+        "aiu",
+        "--compile",
+        "--cast_bf16_to_fp16",
+        "--compile_dynamic",
+        "--min_pad_length",
+        "64",
+        "--max_new_tokens",
+        "5",
+        "--batch_size",
+        str(max_num_seqs),
+        "--compile_dynamic_sendnn",
+        "--attention_type",
+        attn_type,
```

Comment:
This does not occur often, but in cases like this one I would favor readability over consistency and compactness (or line width) in cases like this one to group parameter names and values on one line (if the code formatting police allows it)

```Python
    inference_py_args = [
        sys.executable, "scripts/inference.py",
        "--architecture", "hf_pretrained",
        "--model_path", model_path,
        "--tokenizer", model_path,
        "--unfuse_weights", 
        "--device_type", "aiu",
        "--compile",
        "--cast_bf16_to_fp16", 
        "--compile_dynamic",
        "--min_pad_length", "64",
        "--max_new_tokens", "5",
        "--batch_size", str(max_num_seqs),
        "--compile_dynamic_sendnn",
        "--attention_type", attn_type
```